docs/GraFx-Studio/guides/output/pdf/index.md

```diff
@@ -0,0 +1,24 @@
+# Output to PDF (experimental)
+
+!!! example "Experimental"
+	To give you early access to the latest and greatest, we will release some features as "**Experimental**". This means you can start playing around with this functionality, but you should not use this in production. Functionality might change in the final implementation.
+
+## Select the right Layout
+
+In the tree of layouts, select the one you wish to output.
+
+![Output](output-1.png)
+
+![Output](output-2.png)
+
+## The timeline
+
+Since a PDF is a static file, the timeline does not have any effect on the PDF output. The output engine will disregard all animation information, and generate the PDF as if no animation was made.
```

Comment:
we should specify here if it's the start or end of the timeline that is taken as for the positioning of the items in the template

Reply:
Well, it's neither.
But since that is not clear, we should clarify.
It is the content, without animation. As if you would remove all animation from the timeline.